literature/endo_1999_in-situ_19/endo_1999_in-situ_19_p1_f1b_bold.yaml

```diff
@@ -0,0 +1,53 @@
+# This file is part of echemdb (https://github.com/echemdb/website)
+# Details on the meta data structure can be found on https://github.com/echemdb/metadata-schema
+electrochemical system:
+  electrodes:
+    configuration: 3
+    counter electrode:
+      material: Pt
+      shape: wire
+    reference electrode:
+      type: Ag/AgCl
+    working electrode:
+      crystallographic orientation: '100'
+      material: Ag
+      preparation procedure: Etched with H2O2 and NaCN solution.
+      source:
+        supplier: Techno Chemics
+  electrolyte:
+    components:
+    - concentration:
+        unit: umol / l
+        value: 50
+      name: NaBr
+      type: salt
+    - name: water
+      type: solvent
+    - concentration:
+        unit: mol / l
+        value: 0.05
+      name: NaOH
+      type: base
+    temperature:
+      unit: K
+      value: 298.15
+    type: aq
+  version: 1
+source:
+  citation key: endo_1999_in-situ_19
+  curve: bold
+  url: https://doi.org/10.1016/S0022-0728(99)00148-5
+  figure: 1b
+  version: 1
+version: 1
+curation:
+  version: 1
+  process:
+  - role: curator
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    date: 2021-10-25
+  - role: reviewer
+    name: Albert Engstfeld
+    orcid: https://orcid.org/0000-0002-9686-3948
+    date: '2022-01-21'
```

Comment:
How should the date be formatted here? I guess since YAML supports dates, we should use the version without the quotation marks.

Reply:
I second this.

Reply:
I will change it to the version without quotation. As @saraedum mentions, it would be good to have it in datetime if one does work with the YAML file.
Interestingly, I imported the YAML file with `ruyaml` and added the new datetime entry. When dumping only the existing datetime entry was preserved. I have yet to find out why this is the case. 

In any case, if we find out that this is an issue, we can easily transform all datetimes to a string at a later stage.